preview-src/index.css

```diff
@@ -22,6 +22,11 @@ a:focus, input:focus, select:focus, textarea:focus {
 	border: none;
 }
 
+#timeline-events {
+	padding: 8px 0;
```

Comment:
The `#timeline-events` is for the entire wrapper for comments, commits, and reviews. So the padding gets applied to the beginning and end. In your PR description you only show a stream of commits, however this breaks when you have variations in the timeline:

<img width="571" alt="image" src="https://user-images.githubusercontent.com/35271042/46645508-ff346b80-cb39-11e8-9c1c-a1a8b1c70fe1.png">
<img width="672" alt="image" src="https://user-images.githubusercontent.com/35271042/46645527-1b380d00-cb3a-11e8-84f2-58929eec8b49.png">


I'd probably remove this and tweak the padding/margin on `.comment-container[data-type="commit"]`